connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -817,19 +726,14 @@ public void testAccessors() throws Exception {
         Herder.Created<ConnectorInfo> connectorInfo = createCallback.get(1000L, TimeUnit.SECONDS);
         assertEquals(createdInfo(SourceSink.SOURCE), connectorInfo.result());
 
-        EasyMock.reset(transformer);
-        EasyMock.expect(transformer.transform(eq(CONNECTOR_NAME), EasyMock.anyObject()))
-            .andThrow(new AssertionError("Config transformation should not occur when requesting connector or task info"))
-            .anyTimes();
-        EasyMock.replay(transformer);
-
+        reset(transformer);
         herder.connectors(listConnectorsCb);
         herder.connectorInfo(CONNECTOR_NAME, connectorInfoCb);
         herder.connectorConfig(CONNECTOR_NAME, connectorConfigCb);
         herder.taskConfigs(CONNECTOR_NAME, taskConfigsCb);
         herder.tasksConfig(CONNECTOR_NAME, tasksConfigCb);
-
-        PowerMock.verifyAll();
+        // Config transformation should not occur when requesting connector or task info
+        verify(transformer, never()).transform(eq(CONNECTOR_NAME), any());
```

Comment:
This is much cleaner than the existing strategy (where we cause invocations of `transformer::transform` to throw an exception). Nice job, thanks!